rtl/eclrtl/rtlrecord.hpp

```diff
@@ -25,22 +25,189 @@
 #endif
 
 #include "eclrtl_imp.hpp"
+#include "rtlds_imp.hpp"
 #include "rtlfield.hpp"
 
+//---------------------------------------------------------------------------
+// Record size handling.
+
+class CGlobalHelperClass : public RtlCInterface
```

Comment:
I suspect all these classes now need 
ECLRTL_API
added to the class definition.